examples/apps/generic_client/generic_client.py

```diff
@@ -455,6 +455,17 @@ def Main(args=None):
     # Initializing Worker Object
     worker_obj = worker_details.SGXWorkerDetails()
     worker_obj.load_worker(worker_retrieve_result['result']['details'])
+    verify_key = worker_obj.verification_key
+    encryption_key = worker_obj.encryption_key
+    encryption_key_signature = worker_obj.encryption_key_signature
+
+    # Verify worker encryption key signature using worker's verification key
+    sig_obj = signature.ClientSignature()
+    sig_status = sig_obj.verify_encryption_key_signature(
+        encryption_key_signature, encryption_key, verify_key)
+    if (sig_status != SignatureStatus.PASSED):
+        logger.error("Failed to verify worker encryption key signature")
+        exit(-1)
```

Comment:
Can we move this block to a function `verify_worker_encryption_signature()` and then call this function from ` _do_worker_verification()`

Reply:
Done